modules/ROOT/pages/configuration/plugins.adoc

```diff
@@ -0,0 +1,59 @@
+[[plugins]]
+= Plugins
+:description: This page describes how to load plugins to a Neo4j deployment.
+
+This page describes how to load plugins to a Neo4j deployment.
+The following plugins are supported:
+
+.Supported Neo4j plugins
+[options="header",cols="d,m,b,a"]
+|===
+|Name |Key  | Licence required | Further information
+
+| APOC
+| `apoc`
+| {cross-mark}
+| https://neo4j.com/docs/apoc/current/[APOC user guide]
+
+| Bloom
+| `bloom`
+| {check-mark}
+| link:{neo4j-docs-base-uri}/bloom-user-guide[Neo4j Bloom]
+
+| Streams
+| `streams`
+| {cross-mark}
+| link:{neo4j-docs-base-uri}/kafka-streams[Neo4j Streaming Data Integrations User Guide]
+
+| Graph Data Science
+| `graph-data-science`
+| {cross-mark}
+| link:{neo4j-docs-base-uri}/graph-data-science[Graph Data Science]
+
+| Neo Semantics
+| `n10s`
+| {cross-mark}
+| https://neo4j.com/labs/nsmtx-rdf/[Neo Semantics]
+|===
+
+[NOTE]
+====
+Refer to these other pages on plugins in case you are using link:https://neo4j.com/docs/desktop-manual/current/operations/install-plugin/[Neo4j Desktop], xref:docker/operations.adoc#docker-neo4j-plugins[Docker], or xref:/kubernetes/plugins/[Kubernetes]. 
```

Comment:
I think it is a good practice to also give the names of the pages we are referring to in case the links are broken. For example,
```suggestion
For more information on using plugins in a different Neo4j setup, see link:{neo4j-docs-base-uri}/desktop-manual/current/operations/install-plugin/[Neo4j Desktop -> Install a plugin], xref:docker/operations.adoc#docker-neo4j-plugins[Docker -> Configure Neo4j plugins],  xref:/kubernetes/plugins/[Kubernetes], and link:{neo4j-docs-base-uri}/java-reference/{page-version}/extending-neo4j/project-setup/#_build_dependencies[Java-Reference -> Setting up a plugin project]. 
```